### PR TITLE
use certifi in build

### PR DESCRIPTION
**Pull request by lepelog**
build: https://nightly.link/lepelog/ssrando/actions/runs/1478253756

### CODE REVIEW

File: ssrando.spec [hunk 1]
```diff
@@ -49,6 +49,14 @@ else:
 with open('version-with-git.txt','w') as f:
   f.write(VERSION)
 
+hiddenimports = []
+
+try:
+  import certifi
+  hiddenimports.append('certifi')
+except:
```
Comment (yannl35133) [inline ssrando.spec:57]: Empty excepts look bad, why not have
```suggestion
except ModuleNotFondError:
```

Comment (lepelog) [reply ssrando.spec:57]: It doesn't do any error handling anyways, so I don't see the benefit of adding which error gets thrown here. Plus I don't have to worry about if there is any other error that can possibly be thrown

Comment (yannl35133) [reply ssrando.spec:57]: Catching all errors (if you don't re-raise) is usually bad practice; you don't want to assume all errors should be ignored.
In this case, imagine for some reason importing certifi blocks (eg. infinite loop)
Then, if the user interrupts, the rando resumes in the `except` case, while the user just wanted to completely interrupt everything

**Comment by lepelog** [comment]
didn't seem to help

**Pull request closed without merging**